source/ConditionsStore.h

```diff
@@ -114,11 +112,10 @@ class ConditionsStore {
 	// connected provider).
 	int64_t Get(const std::string &name) const;
 
-	// Add a value to a condition, set a value for a condition or erase a
-	// condition completely. Returns true on success, false on failure.
+	// Add a value to a condition or set a value for a condition. Returns true on
+	// success, false on failure.
```

Comment:
Can we leave the second sentence on its own line so the break isn't mid-sentence?
```suggestion
	// Add a value to a condition or set a value for a condition.
	// Returns true on success, false on failure.
```